FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -314,9 +317,12 @@ - (void)activateWithCompletion:(FIRRemoteConfigActivateChangeCompletion)completi
                                           toSource:RCNDBSourceActive
                                       forNamespace:self->_FIRNamespace];
     strongSelf->_settings.lastApplyTimeInterval = [[NSDate date] timeIntervalSince1970];
+    // New config has been activated at this point
     FIRLogDebug(kFIRLoggerRemoteConfig, @"I-RCN000069", @"Config activated.");
     [strongSelf->_configContent activatePersonalization];
-
+    dispatch_async(dispatch_get_main_queue(), ^{
+      [self notifyConfigHasChanged];
```

Comment:
another thought is should we only send the notification for `firebase` namespace activations? or maybe include the namespace in the notification? i can't really think of a Swift UI use-case for `fireperf` activations..